Typo corrections and some other small fixes (#3362)
iterative/dvc.org: content/docs/user-guide/project-structure/dvcignore-files.md

26ab0db90d72e28ad0ba1e22ee510510  data/data2
```

There are only 2 cache entries, and one of them (the one starting with `ad` and
ending in `.dir`) is for the `data/` directory itself. Checking the hash value
of the data files manually, we can see that the other cache entry (the one
starting with `26`) is for `data2`. There is no cache entry for the `data1` file
(whose hash value starts with `ab`). This means that `dvc add` did ignore

> `data1`.

Reviewer: TMI, I think. This doc is not about cache structure. But we could link to https://dvc.org/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory if needed.

Was there a specific reason for this update?

Data and pipelines are up to date.
```

`dvc status` ignores `data1`.

> Similarly, deleting a dvc ignored file also does not affect `dvc status`:

Reviewer: ```suggestion
Similarly, deleting a file ignored by DVC also does not affect `dvc status`:
```

$ dvc add data
```

> If we move the ignored file to a new file within the `data` directory (which is
> not dvc ignored), DVC will behave as if we modified the directory by adding a

Reviewer: ```suggestion
If we move the ignored file into the `data/` directory (which is
not dvc ignored), DVC will recognize that we modified the directory by adding a
```

```

Only the second file is displayed because DVC now ignores `data1.dvc` and

> `dir1`.

Reviewer: Hmmm this is still confusing. Should be something like "now ignores all files in `dir1/`."
